R/utils.R

```diff
@@ -280,7 +307,11 @@ beautify <- function(x, standardized = FALSE, use_math = FALSE, ...) {
 
   # rearrange ----
   multivariate <- paste0("multivariate.", c("statistic", "df1", "df2"))
-  ordered_cols <- intersect(c("term", "estimate", "conf.int", multivariate, "statistic", "df", "df1", "df2", "p.value"), colnames(x))
+
+  se <- NULL
+  if(!any(colnames(x) == "conf.int")) se <- "std.error"
+
+  ordered_cols <- intersect(c("term", "estimate", "conf.int", se, multivariate, "statistic", "df", "df1", "df2", "p.value"), colnames(x))
```

Comment:
The order of columns should be standardized across methods, right? Could we turn this into a convenience function?

Reply:
Like that idea, will do.

Reply:
done